server/lib/builders/document-builder.js

```diff
@@ -25,7 +25,7 @@ module.exports = exports = class DocumentBuilder {
 		this.content = content;
 
 		this.contentDocument = new DOMParser().parseFromString(
-			`<body>${content.bodyHTML}</body>`,
+			`<body><h3>${content.standfirst}</h3>${content.bodyHTML}</body>`,
```

Comment:
question: is it possible an article won't have a `standfirst`?

Reply:
i don´t know but in that case it going to be parsed to a pdf without contend on the "h3". so i supposed will work anyway and looks ok